packages/react/src/components/Tile/Tile-test.js

```diff
@@ -19,6 +19,14 @@ import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
+const observe = jest.fn();
+const unobserve = jest.fn();
+
+window.ResizeObserver = jest.fn(() => ({
+  observe,
+  unobserve,
+}));
+
```

Comment:
Make sure to do this in a `beforeEach` block and then restore it in `afterEach`, otherwise it may interfere with other tests in the future that expect this to be the native `window.ResizeObserver`.

For example:

```js
let nativeResizeObserver = null;

beforeEach(() => {
  nativeResizeObserver = window.ResizeObserver;
  window.ResizeObserver = jest.fn(() => {
    return {
      observe: jest.fn(),
      unobserve: jest.fn(),
    };
  });
});

afterEach(() => {
  window.ResizeObserver = nativeResiveObserver;
});
```

Another option could be using `jest.spyOn`

```js
let mockObserve;
let mockUnobserve;

beforeEach(() => {
  mockObserve = jest.spyOn(window.ResizeObserver, 'observe').mockImplementation(() => {});
  mockUnobserve = jest.spyOn(window.ResizeObserver, 'unobserve').mockImplementation(() => {});
});

afterEach(() => {
  mockObserve.mockRestore();
  mockUnobserve.mockRestore();
});
```